tests/Tests/ORM/Functional/SecondLevelCacheCountQueriesTest.php

```diff
@@ -89,6 +108,23 @@ public function testInsert(int $cacheUsage): void
         self::assertQueryCountByType('INSERT', 2);
     }
 
+    /**
+    /* @param SupportedCacheUsage $cacheUsage
```

Comment:
Didn't notice this before merging.

```suggestion
    * @param SupportedCacheUsage $cacheUsage
```